admin/src/Membership/MemberBoxSpans.jsx

```diff
@@ -8,14 +8,24 @@ import DateTimeShow from "../Components/DateTimeShow";
 import DateShow from "../Components/DateShow";
 import 'react-day-picker/lib/style.css';
 import MembershipPeriodsInput from "./MembershipPeriodsInput";
+import {get} from '../gateway';
 
 
 class MemberBoxSpans extends React.Component {
 
     constructor(props) {
         super(props);
         this.collection = new Collection({type: Span, url: `/membership/member/${props.match.params.member_id}/spans`, pageSize: 0});
-        this.state = {items: []};
+        this.state = {items: [], pending_labaccess_days: "?"};
+        this.pending_actions = get({url: `/membership/member/${props.match.params.member_id}/pending_actions`}).then((r) => {
+            const sum_pending_labaccess_days = r.data.reduce((acc, value) => {
+            if (value.action.action === "add_labaccess_days")
```

Comment:
Super-hårdkodat 🤷‍♂️

Reply:
@emanuelen5  Finns som konstant här: admin/src/Models/ProductAction.js

Reply:
Tack! Jag lägger in det när jag kommer hem.